tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -46,14 +46,11 @@ class REntry;
 class RNTuple;
 class RNTupleModel;
 
-namespace Detail {
+namespace Internal {
 class RPageSink;
 class RPageSource;
-}
```

Comment:
This header includes `ROOT/RPageStorage.hxx`, so likely these forward-declarations aren't needed